superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -170,6 +172,14 @@ function DashboardList(props: DashboardListProps) {
     );
   }
 
+  const handleBulkDashboardExport = (dashboardsToExport: Dashboard[]) => {
+    const ids = dashboardsToExport.map(({ id }) => id);
+    handleResourceExport('chart', ids, () => {
```

Comment:
is the 'chart' value a typo here for bulk dashboard?

Reply:
Ouch, good catch!